src/plugins/reviewDB/entities/User.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2023 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+export interface ReviewDBUser {
+    lastReviewID: number,
+    type : number // -1 banned, 0 normal, 1 admin
```

Comment:
YOU WILL USE A CONST ENUM GUHHHHH

Reply:
yes

Reply:
explode !

Reply:
```suggestion
    type: number // -1 banned, 0 normal, 1 admin
```
also horror fix extra space

Reply:
> explode !

you will stop writing the most deranged code the world has ever seen

Reply:
stupt linter is supposed to fix these issues, I HATE

eitherway I switched to enums you love